src/components/TransactionHistoryRows/TransactionTag.tsx

```diff
@@ -15,6 +18,15 @@ export const TransactionTag: React.FC<TransactionTagProps> = ({ txDetails, trans
       .some(transfer => !!transfer.id)
   }, [transfersByType])
 
+  const rFoxStakingAsset = useAppSelector(state =>
+    selectAssetById(
+      state,
+      txData?.parser === 'rfox' && txData.type === 'thorchain'
+        ? getStakingAssetId(txData.stakingContract)
+        : '',
+    ),
+  )
+
```

Comment:
blocking: we need either a migration to add stakingContract to all the TXs that are already in the store, or make it optional and not throwing in getStakingAssetId or verifying that stakingContract exist

It makes crashing the app if you go to the RFOX page from a state where you already had TXs, because stakingContract wasn't in the schema previously

Reply:
Good catch, just going to do a tx history migration as is pretty standard for updated tx parsing. 1c2d78f46e92f51cb6ecf5a281f109189ff20e46

Reply:
Perfect! 🔥